src/task.rs

```diff
@@ -2407,3 +2393,14 @@ impl<'a> Runner<'a> {
         Ok(all_ok)
     }
 }
+
+pub fn relative_path(name: &String, cwd: &String) -> String {
+    if name.starts_with(':') || (!name.contains('/') && !name.contains("\\")) {
+        name.to_owned()
+    } else {
+        diff_paths(Path::new(&name), Path::new(cwd))
+            .unwrap()
+            .to_string_lossy()
+            .to_string()
```

Comment:
`to_owned` for consistency?

Reply:
This won't work because string lossy returns `Cow` type.

Reply:
This won't work because string lossy returns `Cow` type.